releng/releng-target/lsp4j.target.target

```diff
@@ -3,11 +3,11 @@
 <target name="org.eclipse.xtext.helios.target" sequenceNumber="0">
 	<locations>
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="false" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/tools/orbit/downloads/2023-03"/>
-			<unit id="com.google.gson" version="2.10.1.v20230109-0753"/>
-			<unit id="com.google.gson.source" version="2.10.1.v20230109-0753"/>
-			<unit id="com.google.guava" version="30.1.0.v20221112-0806"/>
-			<unit id="com.google.guava.source" version="30.1.0.v20221112-0806"/>
+			<repository location="https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2023-09"/>
+			<unit id="com.google.gson" version="2.10.1"/>
```

Comment:
To use the Maven Central version of Gson in OSGi context we needed #738 fixed, so I have gone ahead and completed that too.